code-studio/src/js/progress.js

```diff
@@ -1,4 +1,8 @@
-/* globals dashboard, appOptions  */
+/* globals appOptions  */
+
+var clientState = require('./clientState');
+var StageProgress = require('./components/progress/stage_progress.jsx');
+var CourseProgress = require('./components/progress/course_progress.jsx');
```

Comment:
These two look a little like they're from another change, but the change looks correct/complete to me.  Just double-checking this is intentional?